.golangci.yml

```diff
@@ -60,7 +60,7 @@ linters:
     - exportloopref
     # - forbidigo
     - forcetypeassert
-    - funlen
+    # - funlen
```

Comment:
This is my personal opinion, but I really dislike the funlen linter. I think it makes code worse.

It is actually disabled in [conduit itself](https://github.com/ConduitIO/conduit/blob/main/.golangci.yml#L79)

The others I disabled are also disabled in conduit. The `wrapcheck`, for example, was very annoying when returning context.Err errors. But then, there's this [comment](https://github.com/ConduitIO/conduit/blob/4b66347eb89a2bb8cf641565f7602bf36bc33137/.golangci.yml#L65-L66), so idk.

Maybe we should update the connectors template `.golangci.yml`?
I'm fine following the standard, whatever it fits best.

Reply:
To my knowledge, the linter was introduced into Conduit after the project was initialized and after some work was already done, so it wasn't practical to enable all rules at once. I _guess_ that is the reason `funlen` is disabled in Conduit itself. To me, it's good having as many enabled as possible, and only disabling individual lines/methods where we see it can't be done in a nice way. That's just my thinking, I don't think it's a big thing right now.:)

Reply:
I agree with @hariso on this one. `funlen` is a good linting rule to identify ways to refactor code that may be clearer to future developers. It may not be as important when not collaborating with others on a project, but I'd rather have this rule enabled and disable it only in those situations where, for some reason, we can't get around it.

Reply:
Resolving this as I did enable all them back. I did get around `funlen`, lmk if the new changes fit better.